high_H: 180
high_S: 255
high_V: 255
low_H: 0
low_S: 0
low_V: 0
